### PR TITLE
[fix](complex-column)fix column bitmap with agg behavior

### PR DESCRIPTION
**Pull request by amorynan**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

- Behavior changed:

    - [x] No.
    - [ ] Yes. 

- Does this need documentation?

    - [x] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [x] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by amorynan** [comment]
only branch-2.0 need

**Comment by amorynan** [comment]
run buildall

File: be/src/vec/columns/column_complex.h [hunk 1]
```diff
@@ -81,7 +81,8 @@ class ColumnComplexType final : public COWHelper<IColumn, ColumnComplexType<T>>
         } else if constexpr (std::is_same_v<T, QuantileStateDouble>) {
             pvalue->deserialize(Slice(pos, length));
         } else {
-            LOG(FATAL) << "Unexpected type in column complex";
+            throw doris::Exception(ErrorCode::INTERNAL_ERROR, "Unexpected type in column complex");
+            __builtin_unreachable();
```
Comment (HappenLee) [inline be/src/vec/columns/column_complex.h:85]: the func is void， seems no need add `__builtin_unreachable()`

File: be/src/vec/columns/column_complex.h [hunk 2]
```diff
@@ -205,10 +208,15 @@ class ColumnComplexType final : public COWHelper<IColumn, ColumnComplexType<T>>
...
+        throw doris::Exception(ErrorCode::INTERNAL_ERROR,
+                               "serialize_value_into_arena not implemented");
+        __builtin_unreachable();
     }
 
     [[noreturn]] const char* deserialize_and_insert_from_arena(const char* pos) override {
+        throw doris::Exception(ErrorCode::INTERNAL_ERROR,
+                               "deserialize_and_insert_from_arena not implemented");
+        __builtin_unreachable();
         LOG(FATAL) << "deserialize_and_insert_from_arena not implemented";
```
Comment (HappenLee) [inline be/src/vec/columns/column_complex.h:220]: should we delete the `log(fatal)`

**Comment by amorynan** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**